PLATFORM_SUPPORT.md

```diff
@@ -33,10 +33,22 @@ Community members may provide assistance with these platforms.
 
 |Architecture|Ubuntu Focal (20.04)|MacOS Mojave (10.14)|Windows 10 (VS2019)|Debian Buster (10)|
 |------------|--------------------|--------------------|-------------------|------------------|
-|amd64       |Tier 1 [a][s]       |Tier 1 [s]          |Tier 1 [a][s]      |Tier 3 [s]        |
-|amd32       |                    |                    |Tier 1 [a][s]      |                  |
-|arm64       |Tier 1 [a][s]       |                    |                   |Tier 3 [s]        |
-|arm32       |Tier 3 [s]          |                    |                   |Tier 3 [s]        |
+|amd64       |Tier 1 [^a][^s]     |Tier 1 [^s]         |Tier 1 [^a][^s]    |Tier 3 [^s]       |
+|amd32       |                    |                    |Tier 2 [^a][^s]    |                  |
+|arm64       |Tier 1 [^a][^s]     |                    |                   |Tier 3 [^s]       |
+|arm32       |Tier 3 [^s]         |                    |                   |Tier 3 [^s]       |
 
-" [a] " Binary releases are provided as a single archive per platform.\
-" [s] " Compilation from source.
+[^a]: Binary releases are provided as a single archive per platform.
+[^s]: Compilation from source.
```

Comment:
Can we make these a footnote on the table?

Reply:
I do not think we can do this in an easy way without including some auxiliary package. I will leave as it is for the moment.